interactions/client/bot.pyi

```diff
@@ -51,12 +69,58 @@ class Client:
     async def wait_until_ready(self) -> None: ...
     def event(self, coro: Coroutine, name: Optional[str] = None) -> Callable[..., Any]: ...
     def change_presence(self, presence: ClientPresence) -> None: ...
+    @overload
```

Comment:
We do not need to have numerous `@overload` assignments in our interfacing scheme. This is excessive boilerplate that will make maintaining this later on down the line messier. We're essentially hardcoding type vars, which we're wanting to avoid. My recommendation is ditching `TypeVar` as while it's great for other employments, would not suit a low profile in the library.

Reply:
Additionally, having numerous assignments of this will bring back the problem of having to further maintain it with adding even *more* object types that Discord may potentially introduce in the future.

Reply:
Removing them